README.md

```diff
@@ -8,8 +8,6 @@ enhanced features.
 
 For more information on Myst Online, see http://mystonline.com/developers/
 
-For a project roadmap, see https://github.com/H-uru/Plasma/wiki/Roadmap
```

Comment:
Would be good to add this link on the website's "Get Involved" page instead. I don't think this wiki page (or any part of the wiki...) is linked anywhere else.

Reply:
There's not much left on that page, and we've been trying to move more towards GitHub Discussions for things like roadmap ideas.

There is a lack of pointing to GitHub Discussions on the website "Get Involved" page though, which should probably be fixed...